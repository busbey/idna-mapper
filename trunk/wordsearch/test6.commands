rotate row right 4
