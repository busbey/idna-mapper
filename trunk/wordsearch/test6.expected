abcdefghijklmnop
abcdefghijklmnop
abcdefghijklmnop
abcdefghijklmnop
pabcdefghijklmno
abcdefghijklmnop
abcdefghijklmnop
